docs/open-source-project/porting/system.md

```diff
@@ -100,6 +100,14 @@ To use most of the `resourced` functionalities, you must enable the following `c
 >
 > To use the `resourced` freezer feature, you must install the freezer plugin by enabling `CONFIG_FREEZER`.
 
+Resourced supports several swapping scenarios to provide more RAM to applications running in the foreground. To support them, enable the following options in the kernel configuration:
+
+  - `CONFIG_MEMCG_SWAP`: control swapping applications' memory, `CONFIG_SWAP` will be enabled automatically
```

Comment:
Please rephrase to:

Controls memory of the swapping application. `CONFIG_SWAP` is enabled automatically.

Global Suggestion: For complete sentences, it is recommended to provide a period (.) at the end of the sentence. Please make it consistent through out the document.